frontend/packages/ux-editor/src/components/WithHoverAddButton/WithHoverAddButton.test.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { WithHoverAddButton, type WithHoverAddButtonProps } from './WithHoverAddButton';
+import { BASE_CONTAINER_ID } from 'app-shared/constants';
+import { renderWithProviders } from '@altinn/ux-editor/testing/mocks';
```

Comment:
Nitpicks for relative imports in local package
```suggestion
import { renderWithProviders } from '../../testing/mocks';
```